docs/en/integrations/data-visualization/hashboard-and-clickhouse.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: Hashboard
+sidebar_position: 132
+slug: /en/integrations/hashboard
+keywords: [clickhouse, hashboard, connect, integrate, ui, analytics]
+description: Hashboard is a robust analytics platform that can be easily integrated with ClickHouse for real-time data analysis.
+---
+
+# Connecting ClickHouse to Hashboard
+
+[Hashboard](https://hashboard.com) is an interactive data exploration tool that enables anyone in your organization to track metrics and discover actionable insights. Hashboard issues live SQL queries to your ClickHouse database and is particularly useful for self-serve, adhoc data exploration use cases.  
+
+
+<img src={require('./images/hashboard_01.png').default} class="image" alt="Hashboard data explorer" />  
+
+<br/>
+
+This guide will walk you through the steps to connect Hashboard with your ClickHouse instance. This information is also available on Hashboard's [ClickHouse integration documentation](https://docs.hashboard.com/docs/database-connections/clickhouse).
+
+
+## Pre-requisites
+
+- A ClickHouse database either hosted on your own infrastructure or on [ClickHouse Cloud](https://clickhouse.com/).
+- A [Hashboard account](https://hashboard.com/getAccess) and project.
+
+## Steps to Connect Hashboard to ClickHouse
+
+### 1. Gather Your Connection Details
+
```

Comment:
Could you add `import ConnectionDetails from '@site/docs/en/_snippets/_gather_your_details_native.md';` and `<ConnectionDetails />` snippet? See an example https://raw.githubusercontent.com/ClickHouse/clickhouse-docs/main/docs/en/integrations/data-visualization/grafana-and-clickhouse.md

Reply:
Done 👍